test/functions/iterators/recursive/recursive-iter-findfilesfailure.good

```diff
@@ -1,3 +0,0 @@
-recursive-iter-findfilesfailure.chpl:29: In iterator 'mywalkdirs':
-recursive-iter-findfilesfailure.chpl:39: warning: module-level symbol is hiding function argument 'sort'
-$CHPL_HOME/modules/packages/Sort.chpl:nnnn: note: found 'sort' defined here
```

Comment:
Could you explain why this warning was removed? The test's code seems like a warning would be a appropriate here. (I also don't see mechanically why the warning was removed -- `sort` was brought in via "all contents", and there's another `sort` defining it above.

Reply:
Yes, actually I already explained it in the commit message for 96cb66cf58b11066767273077c46cdb79ca0f7fd . It did take me a minute to figure out why the warning was removed here. Anyway, here is the explanation:

Generally, the new warning replaces the hidden formal warning, but there are two exceptions here:

* The case where a function is found through a use statement but has the same name as a formal is no longer warned about because the new warning does not apply to function lookups (test/functions/iterators/recursive/recursive-iter-findfilesfailure)
    
* The case of a local variable that shadows something from a use statement that in turn shadows a formal is no longer warned about, by the design of the new warning (test/modules/diten/testIfModuleShadowsLocal)

So, that explains mechanically why the warning was removed. I consider this acceptable in the near term & in the Future Work I have

> * consider a similar error for function calls

which would presumably bring this warning back.